### PR TITLE
Updated game

### PR DESCRIPTION
**Pull request by carlfjones**
Please have a look a these updates

### CODE REVIEW

File: unity_game/Assets/Scripts/CannonBallMovement.cs [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+    Rigidbody2D rb2d;
+
+    Dragon target;
+    Vector2 moveDirection;
+    // Start is called before the first frame update
+    void Start()
+    {
+        rb2d = GetComponent<Rigidbody2D>();
+        target = GameObject.FindObjectOfType<Dragon>();
```
Comment (Danielandro) [inline unity_game/Assets/Scripts/CannonBallMovement.cs:17]: I get it

**Review by IndecentDolphin — APPROVED** [review]
Love the new ocean and ghost features. #goldStar

Dragons of the Caribbean 4ever!

Also great use of 10,000 files. We really need to get that .gitignore file working

File: unity_game/Assets/Scripts/CannonBallOne.cs [hunk 2]
```diff
@@ -0,0 +1,30 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class CannonBallOne : MonoBehaviour
+{
+    // Start is called before the first frame update
+    [SerializeField]
+    GameObject arrow;
```
Comment (Danielandro) [inline unity_game/Assets/Scripts/CannonBallOne.cs:9]: Arrrows????

**Review by Danielandro — APPROVED** [review]
Nice and concise commit

**Pull request merged**